aten/src/ATen/templates/TensorMethods.cpp

```diff
@@ -14,7 +14,7 @@ namespace at {
          #name                                                       \
          " but found ",                                              \
          scalar_type());                                             \
-     return this->unsafeGetTensorImpl()->data_ptr_impl<T>();         \
+     return this->unsafeGetTensorImpl()->mutable_data_ptr_impl<T>(); \
```

Comment:
Is this temporary, is that right?

Reply:
Correct. If this eventually is intended to return a const pointer, it will not call the mutable variant.